contracts/src/spaces/facets/membership/MembershipBase.sol

```diff
@@ -50,24 +50,39 @@ abstract contract MembershipBase is IMembershipBase {
   function _collectProtocolFee(
     address buyer,
     uint256 membershipPrice
-  ) internal returns (uint256 protocolFeeBps) {
+  ) internal returns (uint256 protocolFee) {
     MembershipStorage.Layout storage ds = MembershipStorage.layout();
     IPlatformRequirements platform = IPlatformRequirements(ds.spaceFactory);
 
     address currency = ds.membershipCurrency;
     address platformRecipient = platform.getFeeRecipient();
-    uint16 bpsFee = platform.getMembershipBps();
-    protocolFeeBps = BasisPoints.calculate(membershipPrice, bpsFee);
+    protocolFee = _getProtocolFee(membershipPrice);
 
     //transfer the platform fee to the platform fee recipient
     CurrencyTransfer.transferCurrency(
       currency,
       buyer, // from
       platformRecipient, // to
-      protocolFeeBps
+      protocolFee
     );
   }
 
+  function _getProtocolFee(
+    uint256 membershipPrice
+  ) internal view returns (uint256) {
+    MembershipStorage.Layout storage ds = MembershipStorage.layout();
+    IPlatformRequirements platform = IPlatformRequirements(ds.spaceFactory);
+
+    uint256 minPrice = platform.getMembershipMinPrice();
+    uint256 fixedFee = platform.getMembershipFee();
+
+    if (membershipPrice < minPrice && membershipPrice < fixedFee)
+      return fixedFee;
```

Comment:
```suggestion
    if (membershipPrice < minPrice) return fixedFee;
```
If `minPrice <= fixedFee`, this can be simplified. If `minPrice > fixedFee`, the name is misleading.